mautic/mautic
# Fix  save configuration with non existed config parameter

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.0 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:

If you're using `%` in some of configuration parameters, configuration save failed. This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to Configurations > Emails
3. Try insert to Unsubscribed confirmation message  this HTML:

`<style>body {background-color:#f2f2f2;}</style><h1 style="text-align:center;background-color:#fff;margin-left:5%;margin-right:5%;padding:10px;font-size:1.2em;font-weight:normal;">We are sorry to see you go! |EMAIL| will no longer receive emails from us. If this was by mistake, <a href='|URL|'>click here to re-subscribe</a>.</h1>`

4. Save 
5. Without PR there is fatal error

`mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: "The parameter ";margin-right:5" must be defined." at /pathtomautic/var/cache/prod/ContainerX5r38pi/appProdProjectContainer.php line 11156 {"exception":"[object] (Symfony\Component\DependencyInjection\Exception\InvalidArgumentException(code: 0): The parameter ";margin-right:5" must be defined. at /pathtomautic/var/cache/prod/ContainerX5r38pi/appProdProjectContainer.php:11156)"} []`
